tests/nthash.cpp

```diff
@@ -270,33 +292,19 @@ main()
       TEST_ASSERT_EQ(seed_nthashM2_base.roll(), true);
       TEST_ASSERT_EQ(seed_nthashM3_base.roll(), true);
 
-      hashes.push_back({});
-      for (size_t i = 0; i < seed_nthash.get_hash_num(); i++) {
-        const auto hval = seed_nthash.hashes()[i];
-        TEST_ASSERT_EQ(hval, seed_nthashM1.hashes()[i]);
-        TEST_ASSERT_EQ(hval, seed_nthashM2.hashes()[i]);
-        TEST_ASSERT_EQ(hval, seed_nthashM3.hashes()[i]);
-        TEST_ASSERT_EQ(hval, seed_nthashM1_base.hashes()[i]);
-        TEST_ASSERT_EQ(hval, seed_nthashM2_base.hashes()[i]);
-        TEST_ASSERT_EQ(hval, seed_nthashM3_base.hashes()[i]);
-        hashes.back().push_back(hval);
-      }
-
-      if (seed_nthash.get_pos() > 0) {
-        seed_nthash.peek_back();
```

Comment:
Why are the peek tests deleted?

Reply:
Same as https://github.com/bcgsc/btllib/pull/44#discussion_r807379988, restored in https://github.com/bcgsc/btllib/pull/44/commits/34e585d4fb014d05649280a8c86ab5a96332296c.